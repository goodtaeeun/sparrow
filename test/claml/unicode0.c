int main() {
  char x = '\200';
  printf("%u\n", x);
  return 0;
}
