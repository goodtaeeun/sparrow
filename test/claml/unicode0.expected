/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "unicode0.c"
int main(void) {
  char x;

  {
    {
#line 2
      x = (char)4294967168U;
#line 3
      printf("%u\n", (int)x);
    }
#line 4
    return (0);
  }
}
